server.js

```diff
@@ -18,7 +18,7 @@ exports.staticify = staticify
 // Custom dependencies
 const router = require('./app/routes')
 const noCache = require('./app/utils/no-cache')
-const errorHandler = require('./app/middleware/error-handler')
+const errorHandler = require('./app/error-handling/error-handler')
```

Comment:
hmm not sure if we want to move middleware to separate folders.

Reply:
In that case I'll close this PR. No point moving the nunjucks files only.